website/docs/feature-flag-tutorials/use-cases/user-management-access-controls.md

```diff
@@ -0,0 +1,172 @@
+
+## Implement effective auditing in Unleash
+
+For enterprise organizations, auditing is a critical component to various aspects of the software development lifecycle. Audit logs for feature flag management can be part of your overall security and compliance process, which provides a comprehensive view of every action taken within the feature flag management system. Use [Unleash's auditing capabilities](/reference/events) to track critical information for every significant system interaction:
+
+### Auditing user actions
+
+-   Detailed logs of user authentication attempts
+-   Feature flag creation, modification, and deletion
+-   Changes to project configurations
+-   Role and permission modifications
+-   Environment-specific flag updates
+
+### Metadata tracking
+
+-   Precise timestamps for each action
+-   User identity (including email and user ID)
+-   Source IP address
+-   Specific system components affected
+-   Detailed context of each change
+
+The process begins with configuring robust log retention. Financial and healthcare organizations typically require extensive log preservation, maintaining detailed records for up to seven years. For most enterprise environments, a three-year retention period provides a balance between compliance and operational efficiency.
+
+In your **Projects** view, click on your project and select the **Event log** tab to get a comprehensive list of events.
+
+![This event log captures a change request cancellation, the user who performed the cancellation, and other metadata.](/img/use-case-user-mgmt-event-log.png)
+
+Unleash supports multiple log export formats, including JSON and CSV, for integration with enterprise reporting and analytics tools. This flexibility allows security teams to transform raw audit logs into meaningful insights about system usage, user behaviors, and potential security risks.
+
+Here’s an example of an audit log entry in JSON format:
+
+```json
+{
+    "timestamp": "2024-03-15T14:22:11Z",
+    "user": {
+        "id": "user-123",
+        "email": "jane.developer@company.com"
+    },
+    "action": "feature_flag_updated",
+    "details": {
+        "flag_name": "payment-gateway-experiment",
+        "previous_state": "enabled",
+        "new_state": "disabled",
+        "environment": "production"
+    },
+    "source_ip": "192.168.1.100"
+}
+```
+
+The most advanced implementations integrate Unleash audit logs directly into broader Security Information and Event Management (SIEM) systems. This enables real-time monitoring and creates a holistic view of system interactions across multiple platforms. By correlating Unleash audit data with other enterprise security logs, organizations can detect subtle patterns and potential security anomalies that might go unnoticed in isolated systems.
+
+By implementing intelligent, automated, and granular access controls, organizations can create a secure foundation for continuous delivery at scale.
```

Comment:
This conclusion sentence seems out of place.  Only refers to access controls, not the rest of the topics in the blog.